rocketc/rocketc.py

```diff
@@ -446,4 +454,57 @@ def _remove_user_from_group(self, group_name, user_id):
         group = api.search_rocket_chat_group(group_name)
         if group["success"]:
             group = group["group"]
-            api.kick_user_from_group(user_id, group["_id"])
+            return api.kick_user_from_group(user_id, group["_id"])
+        return group
+
+    def get_team_members(self, username):
+        """
+        This method allows to get the members of a team
+        """
+        runtime = self.xmodule_runtime # pylint: disable=no-member
+        course_id = runtime.course_id
+
+        xblock_settings = self.get_xblock_settings()
+
+        try:
+            user = xblock_settings["username"]
+            password = xblock_settings["password"]
+            client_id = xblock_settings["client_id"]
+            client_secret = xblock_settings["client_secret"]
+        except KeyError:
+            raise
+
+        server_url = settings.LMS_ROOT_URL
+
+        api = ApiTeams(user, password, client_id, client_secret, server_url)
+        team = api.get_user_team(course_id, username)
+        if team:
+            team = team[0]
+            team_id = team["id"]
+            members = api.get_members(team_id)
+            if members:
+                for member in members:
+                    yield member["user"]["username"]
+
+    @XBlock.json_handler
+    def leave_group(self, data, suffix=""):
```

Comment:
what groups the user can leave? default course group? team group?

Reply:
You can leave any group except default course group and team group